test/common/index.js

```diff
@@ -6,7 +6,7 @@ const noop = () => {}
 
 const { bin } = require('../../package.json')
 
-process.execPath = path.resolve(__dirname, '..', '..', bin['node-core-test'])
+process.execPath = path.resolve(__dirname, '..', '..', bin.test)
```

Comment:
We could get rid of the magic string here (feel free to ignore, that's going to be useful only in the unlikely event of this pacakge switching name ever again).
```suggestion
const { bin, name } = require('../../package.json')

process.execPath = path.resolve(__dirname, '..', '..', bin[name])
```